jsk_fetch_robot/jsk_fetch_startup/launch/fetch_bringup.launch

```diff
@@ -77,12 +80,16 @@
   <include file="$(find jsk_fetch_startup)/launch/fetch_teleop.xml"
            if="$(arg launch_teleop)" />
 
+  <!-- sound play -->
+  <node name="sound_play" pkg="sound_play" type="soundplay_node.py"
+        respawn="true" if="$(arg launch_sound_play)" />
+
   <!-- speech recognition -->
   <node name="respeaker_transformer" pkg="tf" type="static_transform_publisher"
         args="0 0 0.1 0 0 0 head_pan_link respeaker_base 100"/>
   <include file="$(find julius_ros)/launch/julius.launch">
     <arg name="launch_audio_capture" value="false"/>
-    <arg name="launch_sound_play" value="$(arg launch_sound_play)"/>
+    <arg name="launch_sound_play" value="false"/>
```

Comment:
why you set "false" here? instead of `$(arg launch_sound_play)`

Reply:
we change to launch `sound_play` outside of `julius` launch.
i think this is easier to understand.
https://github.com/jsk-ros-pkg/jsk_robot/pull/1140/files/297f87073395eb72257fde00e0f3fb24c65622e5#diff-f9ac819e72319dde53123b0b9a7ffca8R84-R86